renderer/components/graph.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import Card from './card';
+import { VictoryAxis, VictoryTooltip, VictoryZoomContainer, VictoryChart, VictoryScatter } from 'victory';
+import Tooltip from './tooltip';
+import getFileName from '../../util/getFileName';
+import useTheme from '../hooks/useTheme';
+import { SOUNDS_PER_PAGE } from '../../constants';
+import useAppState from '../hooks/useAppState';
+
+import TsneWorker from 'worker-loader?filename=static/[hash].worker.js!./tsne.worker';
+
+import { Sound } from '../../@types';
+
+const Graph = ({ setPage }: { setPage: (number) => void }) => {
+    const { state } = useAppState();
+    const sounds = state.sounds.data;
+    const nowPlaying = state.sounds.nowPlaying;
+    const [data, setData] = useState([]);
+    const [error, setError] = useState<number>(0);
+    const theme = useTheme();
+    useEffect(() => {
+        setData([]);
+        const names = sounds.map(({ filename }) => getFileName(filename));
+        const ids = sounds.map(({ _id }) => _id);
+        // Missing chroma and MFCC, pitch
```

Comment:
I think those are best left out of this anyways :+1:

Reply:
> Very cool stuff man, nice work on this! In the demo it looks like a border around the canvas is getting cut off, do you know why that is?

The chart library has some weird auto-sizing behaviour and I did not quite fit it 100% into the UI.  Hopefully will get around to fixing it later this week but figured if we wanted to include it in the demo, the code is here.

The tooltip text overlapping the border in the GIF should be fixed.

Reply:
> I think those are best left out of this anyways

Yea I think we could consider converting choma to MIDI or some enumerated value but I don't think we can use pitch and MFCC since its time domain with length varying per recording.